README.md

```diff
@@ -1,6 +1,96 @@
+
+Non-requirements, it does not do these:
+- full compare of past versions, we may wish to add this later
+- reverting to old versions, this was not a requirement for us
+
+## New Features in this fork
+
+## tracking m2m
+- stores changes to m2m lists.  These could not be tracked using existing upstream code.
+- currently requires specifically naming each m2m field to be tracked.
+- stores human-readable changes in "changes" field.  This shows string representation of child table rows added and removed.
+- stores JSON representation of changes in "additional data" field.  This includes ids child table rows added and removed.
+```
+from auditlog.registry_ext import auditlog_register_m2m
+
+class Category(models.Model):
+    name = CharField(max_length=30, null=True)
+
+class Blog(models.Model):
+    categories = ManyToManyField(Category, null=True)
+
+auditlog.register(Blog)
+auditlog_register_m2m(Blog.categories)
+```
+
+Note m2m and mptt tracking results in additional logentry rows.  For example, if you have a model that includes an m2m field, an mptt field, and 3 other fields, 
+
+Because django emits separate signals for model, m2m, and mptt structure changes, we did not see a straighforward way to combine these into a single logentry row.
+
+## tracking mptt structure changes
+- stores previous parent, and new parent.  These could not be tracked using existing upstream code.
+- currently requires specifically naming each mptt field to be tracked.
+- stores human-readable changes in "changes" field.  This shows string representation of node parent before and after moving the mptt node.
+- stores JSON representation of changes in "additional data" field.  This includes ids of node parent before and after moving the mptt node.
+- mptt tracking depends on model_utils FieldTracker
+- mptt tracking required a new change type, "Action.MOVE"
+
+```
+from auditlog.registry_ext import auditlog_register_mptt
+from model_utils import FieldTracker
+
+class MyTree(MPTTModel):
+    parent = TreeForeignKey('self', null=True, blank=True, related_name='children', db_index=True)
+    tracker = FieldTracker()
+
+auditlog.register(MyTree)
+auditlog_register_mptt(MyTree.parent)
+```
+
+## additional_data
+- we assume additional_data is a dict (to store in JSON format)
+- m2m and mptt handlers stuff additional fields into additional_data
+- "changes" field is human-friendly format, whereas "additional_data" field is machine-readable format
+
+## improved admin view
+- for smaller changes, shows complete change right in the changelist view
+- improved change summary in changlist view, including added color coding in changelist view, red for before/deletions, green for after/additions.  Great for quickly reviewing many small changes.
+- exposed more changelist fields and detail view fields, added filters and search fields
+
+## More notes
+
+### integration with django admin change_form UI
```

Comment:
if this feature is not fully supported or requires additional polish it might be better left out of this PR